### PR TITLE
Add feature flags for social links

### PR DESCRIPTION
**Pull request by hannahlspencer**
### What does this PR do?
Adds feature flags for the IQ engine discord and mailing list to differentiate between potential enterprise and community versions.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -73,14 +74,16 @@ export const App = () => {
...
                 </li>
-                <li>
-                  <a href="https://discord.gg/k7C8kp3b76" target="_blank" rel="noreferrer" className="text-lg">
-                    <div className="flex">
-                      <img src="/discord.svg" className="w-6 pr-1" alt="Discord" />
-                      Discord
-                    </div>
-                  </a>
-                </li>
+                <Feature flag="useIQEngineOutReach">
```
Comment (Jaydee123) [inline client/src/App.tsx:77]: would be good to add a component test to test this works when: flag is null, flag is set to true and flag is set to null

File: client/src/App.tsx [hunk 2]
```diff
@@ -203,9 +207,11 @@ export const App = () => {
             rel="noreferrer"
             href="https://dashboard.mailerlite.com/forms/299501/77960409531811734/share"
           >
-            <h2 className="text-center py-2">
-              Sign up for a once-a-month email update on IQEngine, such as new features, demos, and more!
-            </h2>
+            <Feature flag="useIQEngineOutReach">
```
Comment (Jaydee123) [inline client/src/App.tsx:210]: same with this

File: client/src/App.tsx (hunk 1 above)
Comment (Jaydee123) [reply client/src/App.tsx:77]: Also, if feature flag is null, it should be set to true

Comment (Jaydee123) [inline client/src/App.tsx:77]: Should these be individual feature flags? I.e useIQEngineDiscord

**Review by Jaydee123 — APPROVED** [review]
Great work!
